frontend/src/routes/Matches/Match/index.tsx

```diff
@@ -1,16 +1,25 @@
 import React, { useState } from 'react';
 import { useParams } from "react-router-dom";
 import { holePickerPath, matchIdKey } from '../../paths';
-import { useGetMatchStrokesQuery } from '../../../graphql';
+import { useGetMatchStrokesQuery, useSetStrokesMutation } from '../../../graphql';
 import './Match.css'
+import { Console } from 'console';
 
-function StrokePicker({ strokes, setStrokes }: { strokes: number[]; setStrokes: React.Dispatch<React.SetStateAction<any>>; }) {
+function StrokePicker({ strokesId, strokesPar, setStrokes }: { strokesId: string; strokesPar: string; setStrokes: any; }) {
   return (
     <div>
-      {[1, 2, 3, 4, 5, 6, 7, 8, 9].map(stroke => {
+      {[-3, -2, -1, 0, 1, 2, 3].map(strokeVal => {
         return (
-          <button key={stroke} onClick={() => setStrokes([...strokes, stroke])}>
-            {stroke}
+          <button key={strokeVal} onClick={() => 
+            {
+            setStrokes({
+            variables: {
+               strokesId: strokesId,
+               strokes: Number(Number(strokesPar) + Number(strokeVal))
```

Comment:
This looks way more complicated than it needs to be... does `strokesPar + strokesVal` not work?